platforms/jvm/language-java/src/main/java/org/gradle/api/internal/tasks/compile/JavaCompileSpec.java

```diff
@@ -70,4 +70,8 @@ public interface JavaCompileSpec extends JvmLanguageCompileSpec {
     default boolean annotationProcessingConfigured() {
         return !getAnnotationProcessorPath().isEmpty() && !getCompileOptions().getCompilerArgs().contains("-proc:none");
     }
+
+    void setProblemsApiReporting(boolean useProblemsApiReporting);
```

Comment:
Same as above (or below, I don't see ordering in this view :) ).

Reply:
https://github.com/gradle/gradle/pull/27539/files#r1458699522